### DISCUSSION TITLE
Can I change window class？

### DISCUSSION THREAD

**Discussion by USTHzhanglu** [hecrj/iced#2036] [Q&A]
I want use winapi ,so i need get my iced window handle. but when i use FindWindowA by lpWindowName, I always can't get the handle. but successed when by lpClassName. 

![image](https://github.com/iced-rs/iced/assets/56349457/f63b5f8e-0f47-4a6f-aa2e-53e239f45490)

Index by the default classname "Window Class" is so  dangerous. So can i change this by some API?

**Comment by nazar-pc** [comment]
I modified `Tour` example like this to get class set (twice for some reason) to `test` on Linux (X11), probably something similar on Windows:
```rust
    let mut settings = Settings::default();
    settings.window.platform_specific.application_id = "test".to_string();
    Tour::run(settings)
```
